ansys/mapdl/core/inline_functions.py

```diff
@@ -0,0 +1,367 @@
+
+
+class _ParameterParsing:
+    def _parse_parameter_integer_response(self, response) -> int:
+        return int(self._parse_parameter_float_response(response))
+
+    @staticmethod
+    def _parse_parameter_float_response(response) -> float:
+        parts = response.split('=')
+        if len(parts) != 2:
+            raise TypeError('Expression not provided; parameter response not recognised.')
+        number = parts[1].strip()
+        return float(number)
+
+
+class _ComponentQueries(_ParameterParsing):
+    _mapdl = None
+
+    def centrx(self, e: int) -> float:
+        """
+        Fetches centroid X-coordinate of element ``e`` in global
+        Cartesian coordinate system. Centroid is determined from the
+        selected nodes on the element.
```

Comment:
```suggestion
        """Return the x coordinate of the element centroid.

        Fetches centroid X-coordinate of element ``e`` in global
        Cartesian coordinate system. Centroid is determined from the
        selected nodes on the element.
```

Let's get in the habit of providing both a short description and long description following the numpydoc standard.  Reason for this is we're going to be moving towards a documentation style that includes a summary table of all methods and attributes; only the short description will be show (which is expected on the very first line).